integration-tests/pom.xml

```
@@ -59,14 +58,49 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-universe-bom</artifactId>
+            <version>${project.version}</version>
+            <type>pom</type>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <artifactId>maven-invoker-plugin</artifactId>
+                    <version>3.2.1</version>
+                    <configuration>
+                        <projectsDirectory>invoked</projectsDirectory>
+                        <streamLogs>true</streamLogs>
+                        <suppressSummaries>true</suppressSummaries>
+                        <goals>clean install -e</goals>
```

```suggestion
                        <goals>clean verify -e</goals>
```
There is no reason to install, is there?

---

Hm... there seem to be no way to perform the native tests. I think we should define a top level `<invoker.goals>` property so that it is overridable on the CLI - something like `mvn clean verify "-Dinvoker.goals=clean verify -Dnative"`. WDYT?

---

Ah, right, i missed the native args. About verify, I'm not sure it'll work if you clean your local repo.

---

I have sent https://github.com/aloubyansky/quarkus-platform/pull/1 to your fork, @aloubyansky

---

> About verify, I'm not sure it'll work if you clean your local repo.

I can try that tomorrow.

---

So if you want to be able to `mvn clean verify -Dinvoker.test=root/camel-activemq` you need `clean install` as invoker goals. Otherwise, it won't find the rpkgtests artifact.

---

Yeah, I have noticed that. I wonder whether it would be possible to move the rpkg module out of the invoker dir and execute it within the main build.

---

We'd need all the extension test jars configured in the main build then. Not sure, tbh. Needs more evaluation.

---

Thinking more about that, it could certainly be set up, but it is not a priority ATM. I'll do it in the future in case the current setup becomes too limiting for me.